confidential-assets/tests/helpers/index.ts

```diff
@@ -118,7 +118,7 @@ export const mintFungibleTokens = async (account: Account) => {
     const transaction = await aptos.transaction.build.simple({
         sender: account.accountAddress,
         data: {
-            function: `${DEFAULT_CONFIDENTIAL_COIN_MODULE_ADDRESS}::mock_token::mint_to`,
+            function: `${confidentialAsset.confidentialAssetModuleAddress}::mock_token::mint_to`,
```

Comment:
cc @banool, since IIRC you were also modifying this module address stuff

Reply:
this is for a test though

Reply:
This is a good change 👍